docs/_book.yaml

```diff
@@ -74,6 +74,9 @@ upper_tabs:
         path: /cirq/google/calibration
       - title: "Best practices"
         path: /cirq/google/best_practices
+      - heading: "Design proposals"
+      - title: "Request for Comments process"
+        path: /cirq/dev/rfc_process
       - heading: "Developers"
```

Comment:
@lamberta FYI, I'm introducing this as a top level heading. WDYT?

Reply:
LGTM. Eventually you may want a more quantum-wide community section but seems fine to start in Cirq since that's the focus now

Reply:
Thank you!